.travis.yml

```diff
@@ -47,7 +47,7 @@ before_install:
   - if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TEST_SUITE" = "accessibility" ]; then exit 0; fi
   - phpenv config-rm xdebug.ini
   # Lets set-up our helper repository with all the docker config and use correct version in composer.json.
-  - git clone -b 1.0.0 https://github.com/goalgorilla/drupal_social.git install
+  - git clone https://github.com/goalgorilla/drupal_social.git install
```

Comment:
@goalgorilla/maintainers Perhaps we can put back the version (even if it's now `2.x.x` or something) so that it's easier to make breaking changes in our dev setup without immediately bricking our CI.